packages/lit-dev-content/site/tutorials/content/tooltip/00.md

```diff
@@ -0,0 +1,27 @@
+In this tutorial, you'll build a simple tooltip element that supports lazy
+creation, ensuring fast initial page rendering.
```

Comment:
I don't think so. Although it could be a little confusing that the only characteristic of the tool tip you mention is the lazy loading. Could say something like, "... you'll build a tooltip element that supports robust automatic positioning and a simple transition animation. It also supports lazy creation to ensure fast initial page rendering."

Or somethin'